README.md

```diff
@@ -116,7 +117,7 @@ Thanks to @severen for coming up with the name "tealdeer"!
 [c-gh]: https://github.com/tldr-pages/tldr-c-client
 [hs-gh]: https://github.com/psibi/tldr-hs
 [fast-tldr-gh]: https://github.com/gutjuri/fast-tldr
-[bash-gh]: https://4e4.win/tldr
+[bash-gh]: https://github.com/raylee/tldr-sh-client
```

Comment:
For context, it seems like the URL we used previously was changed some time ago: https://github.com/pepa65/tldr-bash-client/commit/6321f7c6df35aa90f3bfb03477f64206cd824d83

However, the client there seems to be unmaintained anyways, so using the one from `raylee` seems sensible. Still, it would also have to be changed in the `Dockerfile`